### PR TITLE
Terraforming

### PR DESCRIPTION
**Pull request by megansam5**
Terraforming the AWS services

### CODE REVIEW

**Comment by MichaelHassan** [comment]
incredible stuff, keep up the good work

File: terraform/.gitignore
Comment (MurasatoNaoya) [inline terraform/.gitignore:1]: Nice accounting for your venv.

File: terraform/README.md
Comment (MurasatoNaoya) [inline terraform/README.md:1]: Solid README for what your terraform is config is building.

File: terraform/main.tf
Comment (MurasatoNaoya) [inline terraform/main.tf:1]: Looks good to me.

**Review by MurasatoNaoya — APPROVED** [review]
Builds all of the cloud infrastructure we need for the project. terraform plan runs as expected also, so locally all seem well.

File: terraform/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+.terraform
+.terraform.lock.hcl
+terraform.tfstate
+terraform.tfstate.backup
+terraform.tfvars
+*.tfvars
+/venv
+venv/
```
Comment (Ruyzambrano) [inline terraform/.gitignore:8]: What about .venv?

File: terraform/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Terraforming
+
+This terraforming creates the lambda for the ETL pipeline, the eventbridge scheduler to run this every minute, the ECS Task definition to run the ECS Task to move the data from short term to long term, and its eventbridge scheduler every 24 hours. And also the task definition for the ECS service for the dashboard.
+
+## Set-up
+
+YOu need to create a `terraform.tfvars` file which includes the following:
+```
+AWS_ACCESS_KEY = "XXX"
+AWS_SECRET_KEY = "XX"
```
[the remark this answers is not in the document]
Comment (megansam5) [reply terraform/README.md:10]: The secret nature is a turn off

**Pull request merged**